doc/migrating-from-legacy.txt

```diff
@@ -0,0 +1,33 @@
+*migrating-from-legacy.txt*   Migrating from legacy-vimfiles
+
+==============================================================================
+MIGRATING FROM LEGACY VIMFILE                             *vimfiles-migrating*
+                                                                   *migrating*
+
+If you're used to the legacy-vimfiles, there are a few things in this vim
+config which may be unfamiliar. Here are the key differences, and why.
+
+-----------------------------------------------------------------------------
+QUICK DIFFERENCES
+
+- Leader has changed from `,` to `<Space>`
+- `:nohl` is no longer bound to `<Space>`. It's now `<Esc><Esc>`
+- NERDTree on `\` is deprecated in favour of |vinegar| and |netrw| on `-`
+  - For a NERDTree-like tree-view in |netrw|, hit `iii`
+- Running tests was `,t`. Now try `<Space>tf`
+
+-----------------------------------------------------------------------------
+PHILOSOPHY
+
+This config (currently available at https://github.com/luan/nvim) is intended
+to be a bit more "pure" than legacy-vimfiles (currently available at
+https://github.com/luan/vimfiles).
+The idea is to not override defaults unless absolutely
+necessary. This should make it easier to migrate back and forth between this
+config, other vim configs, and vanilla vim/vi.
+
+For this reason, features added are exclusively additive to the standard Vim
+experience. For example, while you can disable NERDTree's default hijacking of
+vim's default file browser (netrw) we chose to not include it.
```

Comment:
I think I'd rather mention NERDTree in comparison to vinegar/FZF and not as an example of purity. The decision to not include NERDTree was informed by the hijacking but it mostly spans out of not wanting a tree sidebar thing in the editor.

Reply:
FZF is now `vim-clap` actually